neurom/tests/test_stats.py

```diff
@@ -63,3 +63,9 @@ def test_fit_default_is_normal():
     p1, e1 = st.fit(NORMAL, 'norm')
     nt.assert_items_equal(p0, p1)
     nt.assert_items_equal(e0, e1)
+
+def test_optimal_distribution():
```

Comment:
That's what line 63 is doing for normal. I will add the same for exponential and uniform.